src/k8s-engine/lima.ts

```diff
@@ -966,6 +1008,28 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     });
   }
 
+  /** This shouldn't be lima-specific. We need a mixin */
```

Comment:
```suggestion
  /* This shouldn't be lima-specific. We need a mixin */
```
This is… not a JSDoc :) (We probably should get one though?)

Reply:
done